tbx/project_styleguide/templates/patterns/molecules/streamfield/blocks/image_block.html

```diff
@@ -0,0 +1,20 @@
+{% load wagtailcore_tags wagtailimages_tags %}
+{% image value.image max-1280x860 format-webp as img %}
+<img
+    src="{{ img.url }}"
+    {% if value.image_is_decorative %}
+        alt=""
+    {% else %}
+        alt="{% firstof value.alt img.alt %}"
```

Comment:
Just spotted that this isn't working as expected - in the standard page, if I edit an existing image (e.g. on the oxford geek nights page) and add some alt text, it is still falling back to the image title in the image library

Reply:
Thanks for spotting this @helenb! 

Typo error fixed in 37c88d7